packages/replay/src/coreHandlers/handleKeyboardEvent.ts

```diff
@@ -12,8 +12,9 @@ export function handleKeyboardEvent(replay: ReplayContainer, event: KeyboardEven
     return;
   }
 
-  replay.triggerUserActivity();
```

Comment:
hmm, maybe this... IMHO the "best" solution would be to find the cases where we do not want to capture this (e.g. when tab is involved, ...?) and handle them, but this may be tricky. So probably OK to remove this for now and revisit this later...!